assignments/prototypes.js

```diff
@@ -14,14 +14,28 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
-
+function GameObject(property) {
+  this.createdAt = property.createdAt;
+  this.dimensions = property.dimensions;
+  this.destroy = function() {
+    return `${this.name} was removed from the game`
+  };
+};
 /*
```

Comment:
the destroy function should be outside the GameObject function in the example using a prototype